### DISCUSSION TITLE
How to deal with Page prevented back/forward cache restoration 4 failure reasons

### DISCUSSION THREAD

**Discussion by Lucascn21** [serwist/serwist#95] [Q&A]
Hello
I just picked up the [next basic template](https://github.com/serwist/serwist/tree/main/examples/next-basic) and started it, after running lighthouse it shows the error below:
![image](https://github.com/serwist/serwist/assets/34776035/0a521b36-34a4-4d45-a70e-24736f841ee9)
How do i deal with this issue?

**Comment by DuCanhGH** [comment]
...build it in production mode.
